README.md

```diff
@@ -16,7 +22,12 @@ Documented examples:
 Run `test.sh` to run the tests.
 Files containing tests have to end in `Test`.
 
-The project and all tests run under both Python 2 and 3. You may need to add the `src` directory to your `PYTHONPATH` environment, e.g.
+The project and all tests run under Python 3. You may need to add the
+`src` and `src/SigasiProjectCreator/DotF` directories (the latter for
+`.f` files only) to your `PYTHONPATH` environment, e.g.
 
-* Linux/bash: `export PYTHONPATH=/home/mydir/SigasiProjectCreator/src`
-* Windows: `set PYTHONPATH=C:\work\SigasiProjectCreator\src`
\ No newline at end of file
+* Linux/bash: `export PYTHONPATH=/home/mydir/SigasiProjectCreator/src:/home/mydir/SigasiProjectCreator/src/SigasiProjectCreator/DotF`
+* Windows: `set PYTHONPATH=C:\work\SigasiProjectCreator\src;C:\work\SigasiProjectCreator\src\SigasiProjectCreator\DotF`
+
+For `.f` file parsing, the Antlr4 python runtime must be installed:
+`python3 -m pip install antlr4-python3-runtime`
```

Comment:
We might want to explain 2 paths: with and without antlr?

Reply:
Documentation updated